ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/filter/role/RoleAuditFilter.java

```diff
@@ -126,34 +120,16 @@ private boolean isFilteredUnchecked(AuditEntry logEntry)
 
     private boolean isOperationWhitelistedOnResourceByRoles(Permission operation, List<IResource> operationResourceHierarchy, Set<RoleResource> roles)
     {
-        for (RoleResource role : roles)
-        {
-            if (isOperationWhitelistedOnResourceByRole(operation, operationResourceHierarchy, role))
-            {
-                return true;
-            }
-        }
-
-        return false;
+        return roles.stream()
+                    .anyMatch(role -> isOperationWhitelistedOnResourceByRole(operation, operationResourceHierarchy, role));
     }
 
-    private boolean isOperationWhitelistedOnResourceByRole(Permission operation, List<IResource> operationResourceHierarchy, RoleResource role)
+    private boolean isOperationWhitelistedOnResourceByRole(Permission operation, List<IResource> operationResources, RoleResource role)
```

Comment:
How 'bout operationResourceChain?
Here and on other places where this is passed around?